### ISSUE TITLE
Warning msg in  JCL_Test_Package_0 test

### ISSUE THREAD

**Issue by llxia** [eclipse/openj9#1313]
In JCL_Test_Package test, findPackage() gets @Test annotation added automatically by TestNG. We should make findPackage() private.

Output:
```
Method public static boolean org.openj9.test.java.lang.Test_Package.findPackage(java.lang.String) has a @Test annotation but also a return value: ignoring it. Use <suite allow-return-values="true"> to fix this
```

**Comment by pdbain-ibm** [comment]
I will take this because I am working on that file anyway.
The pull request (in progress) is #579.

**Comment by llxia** [comment]
This issue is fixed by #579.
